gpg.md

```diff
@@ -0,0 +1,116 @@
+GNU Privacy Guard
+=================
+
+![GnuPG](https://upload.wikimedia.org/wikipedia/commons/thumb/6/61/Gnupg_logo.svg/320px-Gnupg_logo.svg.png)
+
+GNU Privacy Guard (GPG) is a complete and free implementation of the OpenPGP
+standard. We use it at Reevoo in order to make it simple and secure to transfer
+sensitive information between people on the team.
+
+Sensitive information includes things like: Passwords, API keys, or customer data.
+
+GPG also allows things like files or git commits to be signed to prove their
+provenance.
+
+## OSX
+
+### Installation 
+
+On OSX the simplest and best integrated way to get GPG is to install
+[GPGtools](https://gpgtools.org/).
+
+This can be done with the package available on the website. Or with
+[Homebrew Cask](https://caskroom.github.io/)
+
+```bash
+brew tap caskroom/cask
+brew cask install gpgtools
+```
+
+### Setup
+
+Once you have installed gpg tools you will need to setup a private key and
+make your public key available to other members of the team.
+
+1. Open the GPG Keychain Application
+  * Click on the New icon in the top left corner
+2. Fill in your details
+   ![Generate Keypair](gpg/gen_keypair.png)
+
+  * Choose a key length of `4096`
+  * Make sure you use a paraphrase to protect your key
+  * Click generate key.
+
+3. Upload Your Key
+   * Choose Preferences from the GPG Keychain Menu
+   ![Keyserver Preferences](gpg/keyserver_pref.png)
+   * Ensure that the keyserver is `hkps://hkps.pool.sks-keyservers.net` 
+   (this should be the default)
+   * `Right Click` > `Send public key to Keyserver`
+   ![Send Public Key](gpg/send_key.png)
+
+4. Wait for your key to become available on the keyserver.
+  * Go to lunch or something, it will take 10 - 15 mins.
+  ![Tea](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b8/Mug_of_Tea.JPG/640px-Mug_of_Tea.JPG)
```

Comment:
Suggested alternative ;) https://imgs.xkcd.com/comics/compiling.png

(only if you want to)